Makefile

```diff
@@ -9,19 +9,17 @@ GOARCH=$(shell go env GOARCH)
 GOHOSTOS=$(shell go env GOHOSTOS)
 GOHOSTARCH=$(shell go env GOHOSTARCH)
 
-CGO_ENABLED=1
-CGO_CFLAGS=-I$(GOPATH)/deps/raft/include/ -I$(GOPATH)/deps/dqlite/include/ -I$(GOPATH)/deps/sqlite/
-CGO_LDFLAGS=-L$(GOPATH)/deps/raft/.libs -L$(GOPATH)/deps/dqlite/.libs/ -L$(GOPATH)/deps/sqlite/.libs/ -L$(GOPATH)/deps/libuv/.libs -L$(GOPATH)/deps/lz4/lib -luv -lraft -ldqlite -llz4
-LD_LIBRARY_PATH=$(GOPATH)/deps/dqlite/.libs/:$(GOPATH)/deps/sqlite/.libs/
-CGO_LDFLAGS_ALLOW="(-Wl,-wrap,pthread_create)|(-Wl,-z,now)"
-
+DEPS_DIR ?= $(PROJECT_DIR)/_deps
 BUILD_DIR ?= $(PROJECT_DIR)/_build
 BIN_DIR ?= ${BUILD_DIR}/${GOOS}_${GOARCH}/bin
 
+CGO_CFLAGS=-I$(GOPATH)/_deps/raft/include/ -I$(GOPATH)/_deps/dqlite/include/ -I$(GOPATH)/_deps/sqlite/
```

Comment:
These should be the `DEPS_DIR` and have the includes.

Reply:
I will push an update for this in a few min.